[BE] minor logging cleanup in distributed

c-p-i-o opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #122831

Summary:
Minor logging cleanup in distributed library
1. Don't use "f" formatted strings - address linter issues.
2. Nits: Make use of unused `e` (error) in a few logs.
3. Change info->debug as asked in issue #113545
4. Nit: rename log -> logger in a few files for consistency

Test Plan:
1. Local build passes.
2. Linter is happy.

Reviewers: wanchaol

Subscribers:

Tasks:

Tags:

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang

c-p-i-o committed:
[BE] minor logging cleanup in distributed

Summary:
Minor logging cleanup in distributed library
1. Don't use "f" formatted strings - address linter issues.
2. Nits: make use of "error" in a few places.
3. Change info->debug as asked in issue #113545
4. Nit: rename log -> logger in a few files for consistency

Test Plan:
1. Local build passes.
2. Linter is happy.

Reviewers: wanchaol

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

wanchaol commented on torch/distributed/elastic/agent/server/local_elastic_agent.py:
```diff
@@ -156,16 +156,16 @@ def _setup_local_watchdog(self, envs: Dict[int, Dict[str, str]]) -> None:
             if watchdog_file_path is None:
                 watchdog_file_path = "/tmp/watchdog_timer_" + str(uuid.uuid4())
-            log.info("Starting a FileTimerServer with %s ...", watchdog_file_path)
+            logger.info("Starting a FileTimerServer with %s ...", watchdog_file_path)
```
should watchdog logging all debug level? cc @wconstab 
I think from a user prospective they don't know what's a watch dog until watchdog brings down processes.

c-p-i-o replied:
I changed the 3 logs in `local_elastic_agent.py` to `debug` but can wait to see what @wconstab might have to see. Have a look and see what you think.

wconstab replied:
well tbh im not even that familiar with _this_ watchdog (the one in elastic, rather than the one in PGNccl).

I'd guess wanchao is right, but @kurman would be more informed.  no objection from me to move to debug here.

kurman replied:
We don't configure it by default, I think info level make sense.

wanchaol commented on torch/distributed/elastic/utils/distributed.py:
```diff
@@ -140,5 +140,5 @@ def get_socket_with_port() -> socket.socket:
         except OSError as e:
             s.close()
-            log.info("Socket creation attempt failed.", exc_info=e)
+            logger.info("Socket creation attempt failed.", exc_info=e)
```
should this be a warn?

wanchaol commented on torch/distributed/fsdp/_flat_param.py:
```diff
@@ -708,7 +708,7 @@ def _init_flat_param_and_metadata(
             and total_numel != total_numel_without_padding
         ):
-            log.info(
+            logger.info(
```
I think this also can be a debug level message as FlatParams is a underlying implementation details that user might not aware of

wanchaol reviewed (approved):
looks great! thanks for the clean up. only have some minor comments

c-p-i-o committed:
Update on "[BE] minor logging cleanup in distributed"


Summary:
Minor logging cleanup in distributed library
1. Don't use "f" formatted strings - address linter issues.
2. Nits: Make use of unused `e` (error) in a few logs.
3. Change info->debug as asked in issue #113545
4. Nit: rename log -> logger in a few files for consistency

Test Plan:
1. Local build passes.
2. Linter is happy.

Reviewers: wanchaol

Subscribers:

Tasks:

Tags:

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang

[ghstack-poisoned]

c-p-i-o committed:
Update

[ghstack-poisoned]

c-p-i-o commented:
@pytorch merge
